compiler/rustc_llvm/build.rs

```diff
@@ -328,6 +329,11 @@ fn main() {
         println!("cargo:rustc-link-lib={kind}={name}");
     }
 
+    println!("cargo:rustc-link-lib=static=LLVMOption");
+    println!("cargo:rustc-link-lib=static=lldCommon");
+    println!("cargo:rustc-link-lib=static=lldELF");
+    println!("cargo:rustc-link-lib=static=lldWasm");
```

Comment:
@whitequark do you know if there is a nicer way to link against lld? `llvm-config` doesn't list it as available component.

Reply:
I think this might be the best way sadly :/